docs/ref/models/fields.txt

```diff
@@ -583,6 +583,12 @@ or :class:`~django.forms.NullBooleanSelect` if :attr:`null=True <Field.null>`.
 The default value of ``BooleanField`` is ``None`` when :attr:`Field.default`
 isn't defined.
 
+.. versionchanged:: 2.1
+
+    In older versions (pre 2.1), this field was ``blank=True`` implicitly.
```

Comment:
Maybe a small rewording:
```
.. versionchanged:: 2.1

    In older versions, this field has :attr:`blank=True <Field.blank>`
    implicitly. You can restore the previous behavior by setting
    ``blank=True``.
```

Reply:
Ok, will change it and submit a PR to stable/2.2.x

Reply:
Thanks :+1:

Reply:
just submitted the new PR /pull/11839. Is it normal to have code failed checks when this is a docs only PR?

Reply:
It depends, sometimes you will hit quite random failures, or failures related with force-push. Don't bother I'll rerun test suite, if necessary.